Index: server.js
===================================================================
diff --git a/server.js b/server.js
--- a/server.js	
+++ b/server.js	
@@ -18,7 +18,17 @@
 app.post('/registration', (req, res) => {
     if (!req.body.email || !req.body.pass)
         return res.sendStatus(400);
-    console.log(req.body);
+    // console.log(req.body);
+    const message = {
+        // from: 'Mailer test <aida35@ethereal.email>',
+        to: req.body.email,
+        subject: 'Congratulations! You are successfully registered our site',
+        text: `your credentials: 
+                login: ${req.body.email}
+                password: ${req.body.pass}`
+
+    }
+    mailer(message);
     user = req.body;
     res.redirect('/registration');
 });
Index: nodemailer.js
===================================================================
diff --git a/nodemailer.js b/nodemailer.js
--- a/nodemailer.js	
+++ b/nodemailer.js	
@@ -1,14 +1,18 @@
 const nodemailer = require('nodemailer');
 
 const transporter = nodemailer.createTransport({
-    host: 'smtp.ethereal.email',
-    port: 587,
-    secure: false,
-    auth: {
-        user: 'aida35@ethereal.email',
-        pass: 'WpCDmXr8Ua546QwaU1'
+        host: 'smtp.mail.ru',
+        port: 465,
+        secure: true,
+        auth: {
+            user: 'maxim_sdrv@mail.ru',
+            pass: 'ryjsCqn1be4vcyarnr89'
+        }
+    },
+    {
+        from: 'Mailer test <maxim_sdrv@mail.ru>'
     }
-});
+);
 
 const mailer = message => {
     transporter.sendMail(message, (err, info) => {
